packages/fast-usdc/src/types.ts

```diff
@@ -55,4 +55,21 @@ export interface PoolMetrics extends PoolStats {
   shareWorth: Ratio;
 }
 
+export interface ChainPolicy {
+  nobleContractAddress: EvmHash;
+  cctpTokenMessengerAddress: EvmHash;
+  confirmations: number;
+  chainName: string;
+  bridgeContractAddress?: EvmHash;
+  eventFilter?: string;
+  chainType?: number;
+  isL2?: boolean;
```

Comment:
```suggestion
  eventFilter?: string;
  chainType?: number;
```

I think we can exclude `bridgeContractAddress` and `isL2` since there are no longer reqs to track L2 -> L1 confirmation.

Reply:
Done